src/tls_wrap.cc

```diff
@@ -353,9 +354,41 @@ Local<Value> TLSWrap::GetSSLError(int status, int* err, std::string* msg) {
         BUF_MEM* mem;
         BIO_get_mem_ptr(bio, &mem);
 
+        Environment* e = env();
+        Isolate* i = e->isolate();
+        Local<Context> c = i->GetCurrentContext();
```

Comment:
We always call these `env`, `isolate`, and `context`, and I would strongly prefer to keep it this way here; `env` can probably just stay `env()`.